src/Features/Onboarding.php

```diff
@@ -521,6 +521,7 @@ public static function get_allowed_plugins() {
 				'klarna-payments-for-woocommerce'     => 'klarna-payments-for-woocommerce/klarna-payments-for-woocommerce.php',
 				'woocommerce-square'                  => 'woocommerce-square/woocommerce-square.php',
 				'woocommerce-shipstation-integration' => 'woocommerce-shipstation-integration/woocommerce-shipstation.php',
+				'woocommerce-payfast-gateway'         => 'woocommerce-payfast-gateway/woocommerce-payfast-gateway.php',
```

Comment:
It looks like the main file path is a bit different for this one which is why the install is failing:

```suggestion
				'woocommerce-payfast-gateway'         => 'woocommerce-payfast-gateway/gateway-payfast.php',
```

Reply:
Thanks @joshuatf, my learning is to just download the plugin zip and have a look :-)